cms/djangoapps/api/v1/tests/test_serializers/test_course_runs.py

```diff
@@ -3,14 +3,14 @@
 
 import datetime
 import ddt
-import pytz
 from django.test import RequestFactory
 
 from common.djangoapps.student.roles import CourseInstructorRole, CourseStaffRole
 from common.djangoapps.student.tests.factories import UserFactory
 from openedx.core.lib.courses import course_image_url
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase  # lint-amnesty, pylint: disable=wrong-import-order
 from xmodule.modulestore.tests.factories import CourseFactory  # lint-amnesty, pylint: disable=wrong-import-order
+from zoneinfo import ZoneInfo
```

Comment:
Sorry, but I just realized that this import was added in a strange place. When imports are split into sections, I think the first is usually from the standard library, the next section (separated by a blank line) are third-party imports, and the final section (also separated by a blank line) are from first-party (or other edx-platform) imports. Does zoneinfo belong at the end of the first group?

If you fix the location of all of these, please do not squash and ensure there is a separate commit, so I can just review that change. Thank you.